modules/contributor-guide/partials/proc_testing-che-theia-with-custom-branding.adoc

```diff
@@ -23,7 +23,7 @@ To test a custom Che-Theia image, create a new `meta.yaml` file describing a cus
 
 . Create a workspace using the sample https://github.com/che-samples/che-theia-branding-example/blob/master/devfile.yaml[che-theia-branding-example devfile] to apply the changes:
 +
-image::https://www.eclipse.org/che/contribute.svg[link="https://che.openshift.io/f?url=https://raw.githubusercontent.com/che-samples/che-theia-branding-example/master/devfile.yaml"]
+image::https://www.eclipse.org/che/contribute.svg[link="https://workspaces.openshift.com/f?url=https://raw.githubusercontent.com/che-samples/che-theia-branding-example/master/devfile.yaml"]
```

Comment:
@benoitf @vitaliy-guliy not sure who is the contact person, but the image in the devfile should be updated to quai 
`docker.io/vgulyy/che-theia-codeready-workspaces:next"` 
```
 'Failed to pull image "docker.io/vgulyy/che-theia-codeready-workspaces:next": rpc error: code = Unknown desc = Error reading manifest next in docker.io/vgulyy/che-theia-codeready-workspaces: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit', 'workspaceicl3lz98sdpnf1wc.che-workspace-pod-6f797bb889-w29m7'
```

Reply:
I will take a look at it

Reply:
I have also this error:


> Error when starting workspace
>
> Unable to start workspace. Error when trying to start the workspace: Unrecoverable event occurred: 'Failed', 'Failed to pull image "docker.io/vgulyy/che-theia-codeready-workspaces:next": rpc error: code = Unknown desc = Error reading manifest next in docker.io/vgulyy/che-theia-codeready-workspaces: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit', 'workspacem8i1w1vcorki3lk1.che-workspace-pod-6b7ff47447-65p6b'
```

Reply:
@vitaliy-guliy please migrate images to quay.io

Reply:
in general the usage of the personal repos should be prohibited e.g. `docker.io/vgulyy/che-theia-codeready-workspaces:next`